css/style.css

```diff
@@ -0,0 +1,903 @@
+    opacity: 0.61;
+    font-family: Merriweather;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 0.0699em;
+    color: #000000;
+    margin-top: 11px;
+    margin-left: -21px;
+}
+
+.first-par,
+.second-par {
+    opacity: 0.74;
+    font-family: Merriweather;
+    font-size: 20px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.068em;
+    color: #000000;
+}
+
+.first-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.07em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.second-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.first-par__text-second {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 0px;
+}
+
+.second-par {
+    letter-spacing: -0.036em;
+}
+
+.second-par_text {
+    margin-top: 13px;
+    margin-left: 1px;
+    letter-spacing: 0.0445em;
+}
+
+.main-image__style {
+    padding-left: 15px;
+}
+
+.main-information__text {
+    width: 474px;
+    padding-left: 2.3%;
+}
+
+.main-information__text-second {
+    margin-top: 29px;
+    margin-left: -2px;
+}
+
+.featured-works {
+    min-height: 1165px;
+    padding-top: 204px;
+}
+
+.featured-works__text {
+    text-align: center;
+    margin-left: 45px;
+}
+
+.featured-works__text-intro {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    color: #000000;
+}
+
+.featured-works__text-main {
+    opacity: 0.61;
+    font-family: Merriweather;
+    font-size: 17px;
+    line-height: 28px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 0.072em;
+    color: #000000;
+    margin-top: 9px;
+    margin-left: -41px;
+}
+
+.popup {
+    display: flex;
+    position: fixed;
+    align-items: center;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    background-color: rgba(0, 0, 0, .5);
+    cursor: pointer;
+}
+
+.popup-img {
+    position: absolute;
+    top: 6%;
+    left: 30%;
+    width: 700px;
+}
+
+.hiden-popup {
+    display: none;
+}
+
+.show-popup {
+    display: flex;
+}
+
+.featured-works__images {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 115px;
+    justify-content: space-between;
+    row-gap: 13px;
+    /* margin-left: 19px; */
+}
+
+.featured-img__style {
+    width: 371px;
+    cursor: pointer;
+    transition: .3s;
+}
+
+.featured-img__style:hover {
+    transition: .3s;
+    transform: scale(1.03);

```

Comment:
let's use scss instead of css (https://sass-scss.ru/guide/)

Reply:
Thanks, i understand, but i removed this class